chitterAPI.test.js

```diff
@@ -0,0 +1,51 @@
+require('jest-fetch-mock').enableMocks();
+const ChitterAPI = require('./chitterAPI');
+
+let api;
+
+describe('ChitterAPI', () => {
+  beforeEach(() => {
+    api = new ChitterAPI();
+  })
+
+  describe('loadPosts', () => {
+    it('fetches notes from the server', async () => {
+      let samplePosts = ['Peep', 'Peep2'];
+
+      fetch.mockResponseOnce(JSON.stringify({
+        samplePosts
```

Comment:
Seems pretty straightforward to include the sample posts directly rather that setting them to a variable used only once